src/rpc/signaling-exchange.ts

```diff
@@ -242,8 +242,9 @@ export class SignalingExchange {
     try {
       await this.pc.addIceCandidate(cand);
     } catch (error) {
+      console.log('error adding ice candidate', error); // eslint-disable-line no-console
```

Comment:
This will report an error if this ever happens again and bubble up to the reason of dial failure instead of undefined.